integration/util/kernel.go

```diff
@@ -0,0 +1,18 @@
+package util
+
+import (
+	"github.com/docker/docker/pkg/parsers/kernel"
+)
+
+// MustKernelCmp is a "math/big"-style Cmp function. It compares the current
+// kernel version (lval) to and rval, and returns:
+//   -1 if lval < rval
+//    0 if lval = rval
+//    1 if lval > rval
+func MustKernelCmp(rval kernel.VersionInfo) int {
```

Comment:
Could this be inlined into `run_test.go`? Do you expect to use it outside of container tests?